<launch>

    <param name="config_file" type="string" value="$(find uwb_system)/config/uwb_system_fl.yaml" />

    <!-- visualization -->
    <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find uwb_system)/config/uwb_system.rviz" />
    <node name="pcd_to_pointcloud" pkg="pcl_ros" type="pcd_to_pointcloud" output="screen" args="/home/xr/Documents/research/slam/uwb-cal/data/map_downsampled.pcd 5.0 _frame_id:=camera_init" />

</launch>

